src/components/Topics.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect, useState } from "react";
+import { Link, Route, Routes } from "react-router-dom";
+import { getArticles } from "../api";
+
+export const Topics = () => {
+  const [isLoading, setIsLoading] = useState(false);
```

Comment:
isLoading is never used, let's add some conditional rendering

Reply:
Nice job adding this 👍